src/OOVPADatabase/D3D8LTCG/5849.inl

```diff
@@ -185,6 +185,31 @@ OOVPA_SIG_MATCH(
     //
 );
 
+// ******************************************************************
+// * D3DDevice_MakeSpace
+// ******************************************************************
+OOVPA_SIG_HEADER_XREF(D3DDevice_MakeSpace_0,
+                      5849,
+                      XRefOne)
+OOVPA_SIG_MATCH(
+
+    // call D3D::MakeRequestedSpace
+    XREF_ENTRY(0x0B, XREF_D3D_MakeRequestedSpace),
+
+    // D3DDevice_MakeSpace+0x00 : mov eax,[addr]
```

Comment:
nit: Replace underline to double colons for define class and function. Add space after comma.

I am willing to reformat these comments to use our current format regardless older signatures aren't following the format we expected. In a separate commit of course.